### PR TITLE
test(ticdc): add processor etcd worker delay fail point

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: ref #5326 

### What is changed and how it works?
add processor etcd worker delay fail point

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

File: pkg/orchestrator/etcd_worker.go [hunk 1]
```diff
@@ -42,6 +43,11 @@ const (
 	// takes more than etcdWorkerLogsWarnDuration, it will print a log
 	etcdWorkerLogsWarnDuration = 1 * time.Second
 	deletionCounterKey         = "/meta/ticdc-delete-etcd-key-count"
+
+	// ProcessorRole is the role of the processor etcd worker
```
Comment (hi-rustin) [inline pkg/orchestrator/etcd_worker.go:47]: We already have these values. See: `pkg/util/identify`.

/hold

**Review by hi-rustin — COMMENTED** [review]
A tiny issue.

File: pkg/orchestrator/etcd_worker.go (hunk 1 above)
Comment (sdojjy) [reply pkg/orchestrator/etcd_worker.go]: fixed

**Pull request merged**